src/app/(main)/profile/myExperiences/page.tsx

```
@@ -6,61 +6,94 @@ import ExperienceCard from '@/components/profile/ExperienceCard';
-  const [isLoading, setIsLoading] = useState(false);
+
+  // 🎯 무한 스크롤 관련 상태들
+  const [cursorId, setCursorId] = useState<number | null>(null); // 다음 페이지 시작점 (마지막으로 로드된 체험의 ID)
+  const [hasMore, setHasMore] = useState(true); // 더 로드할 데이터가 있는지 여부
+  const [isLoading, setIsLoading] = useState(false); // 추가 로딩 중인지 (무한 스크롤용)
+  const [isInitialLoading, setIsInitialLoading] = useState(true); // 첫 페이지 로딩 중인지
+
+  // 🎯 무한 스크롤 트리거 요소를 참조하는 ref
   const loaderRef = useRef<HTMLDivElement | null>(null);
+
   const mobileContext = useContext(ProfileMobileContext);
```

ESLint

[error] 'mobileContext' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)